includes/class-sensei-analysis-course-list-table.php

```diff
@@ -328,7 +328,7 @@ public function generate_report( $report ) {
 		$this->search = $search;
 
 		$args = array(
-			'number'  => '',
+			'number'  => -1,
```

Comment:
@donnapep 
Here, I removed the limit itself to allow export all existing data (even if there are 100000 rows). This is how I get “Export all rows”.
On the other hand, we can read “Export all rows” as “Export all visible rows”. In this case, we need to apply the same limit we use on the page.